include/common/qrack_types.hpp

```diff
@@ -27,7 +27,8 @@
 #define ZERO_R1 0.0f
 #define ONE_R1 1.0f
 #define PI_R1 (real1) M_PI
-#define min_norm 1e-9f
+#define min_norm 1e-16f
```

Comment:
Can you provide any magic commentary as to why these magic values, or what changing them might indicate?  Would be a useful comment for the future in the code.

Reply:
Added.